src/screens/profile/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useRef } from 'react';
+import { ScrollView, View } from 'react-native';
+
+import { MainButton, Text } from 'src/components';
+import { AppColor, ButtonMode } from 'src/common/enums';
+import { NAVBAR_BUTTON_WIDTH } from 'src/screens/profile/components/navbar/constants';
+import styles from './styles';
+
+type NavbarProps = {
+  activeIndex: number;
+  onClick: (index: number) => void;
+  items: string[];
```

Comment:
Let's keep methods after properties
```suggestion
  items: string[];
  onClick: (index: number) => void;
```